pages/blog/the-reference-rabbit-hole.md

```diff
@@ -0,0 +1,139 @@
+---
+title: "The Reference Rabbit Hole"
+date: 2021-11-15T12:00:00+01:00
+type: Engineering
+tags:
+  - Specification
+  - JSON Schema 
+cover: /img/posts/the-reference-rabbit-hole.webp
+authors:
+  - name: Jonas Lagoni
+    photo: /img/avatars/jonaslagoni.webp
+    link: https://github.com/jonaslagoni
+    byline: AsyncAPI Maintainer
+---
+
+So [Sergio](https://github.com/smoya) and I, went down a little bit of a rabbit hole the last couple of days when we were discussing [Fran's proposal to solve publish/subscribe confusion](https://github.com/asyncapi/spec/issues/618), and I thought I would share the journey.
```

Comment:
Don't tell me why but it sounds better to me. Obviously, you can skip it because my proficiency in English is meh...

```suggestion
So [Sergio](https://github.com/smoya) and I, went a little bit down a rabbit hole. the last couple of days when we were discussing [Fran's proposal to solve publish/subscribe confusion](https://github.com/asyncapi/spec/issues/618), and I thought I would share the journey.
```

Reply:
Hmmmm 🤔 

```suggestion
So [Sergio](https://github.com/smoya) and I went a little bit down a rabbit hole the last couple of days when we were discussing [Fran's proposal to solve publish/subscribe confusion](https://github.com/asyncapi/spec/issues/618), and I thought I would share the journey.
```

Reply:
I think it is also good for reading 👍